jitbuilder/CMakeLists.txt

```diff
@@ -53,6 +53,7 @@ elseif(OMR_ARCH_POWER)
 		p/runtime/CodeSync.cpp
 		p/runtime/AsmUtil.spp
 		p/runtime/CodeDispatch.spp
+		${OMR_ROOT}/compiler/p/env/OMRDebugEnv.cpp
```

Comment:
This should be added to `compiler/p/CMakeLists.txt`, as it's shared across projects.

Reply:
This is how compilertest does it. I would prefer to keep both products working the same way.  If that is the correct way then compiletest should also be changed.  I would prefer to make a follow on PR to change both if that is the decision.

Reply:
KK.